### ISSUE TITLE
Python 3.12 & pip

### ISSUE THREAD

**Issue by Torxed** [Scille/winfspy#40]
Hi.

Could we get a new **pip** release that targets `Python 3.12.3` or at least `>=3.12`?
It's the latest stable and the current pip release does not allow for targeting 3.12 on windows:

```
$ pip download --verbose --python-version=3.12.3 --only-binary=":all:" --platform win_amd64 --dest ~/Downloads/ "winfspy==0.8.4"
```

Could skip using `--only-binary` and use the source to get around it. But that could put additional requirements on the target system installing the package so having the option to get the binary would be nice.

tl;dr: Uploading a `py312`/`cp312` compatible `*.whl` to the current tagged realease
